src/features/expressions/index.ts

```diff
@@ -546,6 +546,16 @@ export const ExprDefaultsForComponent: ExprDefaultValues<ILayoutComponent> = {
   readOnly: false,
   required: false,
   hidden: false,
+  textResourceBindings: {
+    title: '',
+    tableTitle: '',
+    description: '',
+    body: '',
+    next: '',
+    back: '',
+    altTextImg: '',
+    help: '',
```

Comment:
I guess this works in a pinch, but this would cause a failing expression in `title` to produce a warning and default to an empty string, while a failing expression in `shortName` would cause a full app crash. I think this warrants a possible alternative to `useExpression` that provides a default for every possible text resource binding. I can help with that, if you'd like.

Reply:
Yes, i was trying to find all the text resource bindings that are being used, but i guess i missed some 😅 As for how we should handle the values, do you think we need to consider language selection? Or would it be fine to just use English? If we need to use the selected language i guess we will need to find a good place to resolve the texts. Is that what you mean with an alternative to useExpression? Help would be nice 😄

Reply:
I'll look into it ASAP! And no, I don't think we should complicate it by actually looking up the language keys in text resources. Instead of describing in detail what I'm envisioning, I'll just commit to this branch. :wink:

Reply:
Turned out a little differently (no need for using anything other than `useExpressions()` after all). Pushed now.